.gitignore

```diff
@@ -118,3 +118,7 @@ env/
 
 ### Configuration files ###
 .env
+
+### Firefox webdriver files - should download latest ###
+geckodriver.log
+geckodriver.log
```

Comment:
You could ignore geckodriver.exe as well since your README has installation instructions